servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpExecutionContext.java

```diff
@@ -17,17 +17,14 @@
 
 import io.servicetalk.buffer.api.BufferAllocator;
 import io.servicetalk.concurrent.api.Executor;
+import io.servicetalk.transport.api.DefaultExecutionContext;
 import io.servicetalk.transport.api.IoExecutor;
 
 /**
  * Default implementation of {@link HttpExecutionContext}.
  */
-public final class DefaultHttpExecutionContext implements HttpExecutionContext {
-
-    private final BufferAllocator bufferAllocator;
-    private final IoExecutor ioExecutor;
-    private final Executor executor;
-    private final HttpExecutionStrategy executionStrategy;
+public final class DefaultHttpExecutionContext extends DefaultExecutionContext<HttpExecutionStrategy>
```

Comment:
Was this change made only to reduce duplication or for some other reason?

Reply:
Yes, to reduce duplication. I'd actually prefer that `DefaultHttpExecutionContext` not even exist since it is just a non-generic version of `DefaultExecutionContext<HttpExecutionStrategy>`